### DISCUSSION TITLE
LCR 170. 交易逆序对的总数

### DISCUSSION THREAD

**Discussion by giscus** [itcharge/AlgoNote#190] [Comments]
# LCR 170. 交易逆序对的总数

[https://algo.itcharge.cn/solutions/LCR/shu-zu-zhong-de-ni-xu-dui-lcof/](https://algo.itcharge.cn/solutions/LCR/shu-zu-zhong-de-ni-xu-dui-lcof/)

**Comment by changdq** [comment]
归并排序的merge过程，如果left最后有剩余，应该继续累积逆序对

**Reply by LeBronlJames** [reply]
并不需要，因为它算的时候已经考虑到了

**Reply by changdq** [reply]
让我再考虑一下
